Makefile

```diff
@@ -275,6 +275,7 @@ k8s-client-gen: client-gen lister-gen informer-gen ## Generate typed client for
 
 validate-swagger: ## Validate swagger
 	$(DOCKER) run --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml:z -v $(PWD)/swagger.yaml:/tmp/swagger.yaml:z stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml
+	$(DOCKER) run --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml:z -v $(PWD)/swagger-backend.yaml:/tmp/swagger-backend.yaml:z stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml
```

Comment:
iinw, the path (the last argument) should be changed to `/tmp/swagger-backend.yaml` which is the path to which the yaml file is mounted in the image?

another suggestion is to run validation for both files within the same container. spectral usage should support several files as input.
```
spectral <command>

Commands:
  spectral lint [documents..]  lint JSON/YAML documents from files or URLs
```

Reply:
Done.